dashboard/app/templates.html

```diff
@@ -83,7 +83,9 @@ <h1><a href="/{{$.Namespace}}">syzbot</a></h1>
 					<a class="navigation_tab{{if eq .URLPath (printf "/%v/graph/fuzzing" $.Namespace)}}_selected{{end}}" href='/{{$.Namespace}}/graph/fuzzing'>
 					  <span style="color:DarkOrange;">📈</span> Fuzzing</a>
 					<a class="navigation_tab{{if eq .URLPath (printf "/%v/graph/crashes" $.Namespace)}}_selected{{end}}" href='/{{$.Namespace}}/graph/crashes'>
-						<span style="color:DarkOrange;">📈</span> Crashes</a> 
+						<span style="color:DarkOrange;">📈</span> Crashes</a>
+          <a class="navigation_tab{{if eq .URLPath (printf "/%v/graph/crashes" $.Namespace)}}_selected{{end}}" href='/{{$.Namespace}}/graph/coverage'>
```

Comment:
Idea for a further improvement (we can skip it in this PR): we can cache whether we have any coverage statistics for a namespace and ony show the link for those that have it.